### PR TITLE
NXP-32286: Add configurable ServiceAccount annotations

### PR DESCRIPTION
**Pull request by nichiporenko**

### CODE REVIEW

**Review by jjliggett — DISMISSED** [review]
Thanks!!

File: nuxeo/templates/serviceaccount.yaml [hunk 1]
```diff
@@ -4,4 +4,8 @@ kind: ServiceAccount
 metadata:
   name: {{ include "nuxeo.serviceAccountName" . }}
   labels: {{- include "nuxeo.labels" . | nindent 4 }}
-{{- end -}}
+  {{- with .Values.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+{{- end -}}
```
Comment (efge) [inline nuxeo/templates/serviceaccount.yaml:11]: Please add a newline on the last line

Comment (nichiporenko) [reply nuxeo/templates/serviceaccount.yaml]: Thank you. Added

**Comment by efge** [comment]
(I'd rather let @jjliggett re-approve given I don't know this area)

**Review by jjliggett — DISMISSED** [review]

**Review by ataillefer — APPROVED** [review]

**Review by jjliggett — APPROVED** [review]

**Pull request merged**